website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Operator Observability Best Practices"
+linkTitle: "Observability Best practices"
+weight: 6
+description: This guide describes the best practices concepts for adding Observability to operators.
+---
+
+## Operator Observability Best Practices
+
+In this document we will outline what operators require in order to meet the "Deep Insights" capability level and provide best practices and examples for creating metrics, recording rules and alerts. It is based on the general guidelines in [Operator Capability Levels](https://sdk.operatorframework.io/docs/overview/operator-capabilities/).
+
```

Comment:
This doc only speaks about metrics and alerts. Am I right?
So, I think the title here and overview needs to be specific to metrics and alerts. 

OR 

We need to cover all aspects of how to achieve the level which is not only about metrics and alerts, see: https://sdk.operatorframework.io/docs/overview/operator-capabilities/